docs/dev_guide/API-Mapping-query-guide.rst

```diff
@@ -0,0 +1,34 @@
+API-Mapping Query Guide
+=======================
+
+|tool_name| provides the functionality for users to query functionally
+compatible SYCL API for some CUDA API. The API query functionality can assist
+users on manual code migration and help users understand the SYCL API.
+
+API mapping query example:
+
+   .. code-block:: bash
+
+      $ dpct --query-api-mapping=cudaMalloc
+      CUDA API:
+        cudaMalloc(pDev /*void ***/, s /*size_t*/);
+      Is migrated to:
+        *pDev = (void *)sycl::malloc_device(s, dpct::get_in_order_queue());
+
+- You can use ``tab`` button to auto-complete the CUDA API names (In Linux OS only).

```

Comment:
Change to "On a Linux OS, you can use the ``tab`` button" to auto-complete the CUDA API names."